### PR TITLE
Tabbed interface for user profile

### PR DESCRIPTION
**Pull request by YoungSong99**
Features:
- add nav tab to user page
- create pages for liked, following, followers.

### CODE REVIEW

File: app/views/shared/_profile_nav.html.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!-- shared/_profile_nav.html.erb -->
```
Comment (ghost) [inline app/views/shared/_profile_nav.html.erb:1]: Ohhhh, I like that you made this a partial!! I shoulda done that!!

**Review by ghost — APPROVED** [review]
Looks good! I'll have to go back and add that _profile_nav partial to my code later!

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -3,4 +3,11 @@ def show
     @user = User.find_by!(username: params.fetch(:username))
   end
```
Comment (CarlosCastanedaDev) [inline app/controllers/users_controller.rb:5]: Hi Young, you seem to be missing the liked and feed actions in this controller. Can you please add these.

**Review by CarlosCastanedaDev — APPROVED** [review]
Looks good!

**Pull request merged**